### 4/1/2022

We have a standard (52 card) deck of cards, with equal share of the cards associated with each of the four "suits" (e.g. clubs, spades, diamonds, and hearts).

If you draw 3 cards from the deck, one at a time, what is the probability that you draw a club, a heart, and a diamond in that order? 

$~$

**Solution:**  


If we have a deck of 52 cards with an equal share of the cards associated with each of the four suits, there are 13 cards per suit.  Assuming we draw cards from the deck without replacement, with each draw the number of cards of suit S will decrease by 1, as will the total number of cards in the deck.  

$~$

Therefore:   

The probability of drawing a club first, ${P(\text{club } 1^{\text{st}})}$ is $\frac{13}{52} = \frac{1}{4}$,  

The probabilty of drawing a heart second, ${P(\text{heart } 2^{\text{nd}})}$ is $\frac{13}{52-1} = \frac{13}{51}$, and  

The probabilty of drawing a diamond third, ${P(\text{diamond } 3^{\text{rd}})}$ is $\frac{13}{52-1-1} = \frac{13}{50}$  

$~$

We can multiply the probabilty of these events occurring to get the total probabilty.  

$P(\text{club } 1^{\text{st}} \cap \text{heart } 2^{\text{nd}} \cap \text{diamond } 3^{\text{rd}} ) = P(\text{club } 1^{\text{st}}) \times P(\text{heart } 2^{\text{nd}}) \times P(\text{diamond } 3^{\text{rd}})) = \frac{1}{4} \times \frac{13}{51} \times \frac{13}{50}$

In [1]:
(1/4)*(13/51)*(13/50)

0.01656862745098039

**The probabilty of drawing a club, a heart, and a diamond, in that order, is $\approx 0.0166$**